sklearn/ensemble/tests/test_gradient_boosting.py

```diff
@@ -212,10 +212,14 @@ def test_classification_synthetic(loss):
     check_classification_synthetic(loss)
 
 
-def check_boston(loss, subsample):
-    # Check consistency on dataset boston house prices with least squares
+def check_california(loss, subsample):
```

Comment:
`fetch_california_housing` will require network and it will fail in some case (for instance for people packaging in debian) if we don't put the decorator the `pytest.mark.network.

It might be interested to include a subset of `california_housing` directly in the package for testing purposes.